LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/scroll-target-margin-005.html

```diff
@@ -22,9 +22,12 @@
 </div>
 
 <script>
-test(function() {
-  document.scrollingElement.scrollTo(0, 20000);
-  document.getElementById("target").focus();
+promise_test(async function() {
+   document.scrollingElement.scrollTo(0, 20000);
+   await new Promise(resolve => {
+     document.addEventListener("scroll", () => step_timeout(resolve, 0));
+     document.getElementById("target").focus();
+   });
   // Should be around 4900 (5000 - 200px of margin). Give some leeway to account for line height / borders / input padding / etc.
   assert_between_exclusive(document.scrollingElement.scrollTop, 4750, 4850, "Should honor date input scroll-margin");
```

Comment:
Do we want to adjust this interval?

Reply:
Yes, chrome is failing by a similar margin so I think its fine to increase the leeway a bit.